### PR TITLE
Fix todo from #1765

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: src/widget/woverview.cpp [hunk 1]
```diff
@@ -340,7 +337,7 @@ void WOverview::onMarkRangeChange(double /*v*/) {
     update();
 }
 
-void WOverview::onRateSliderChange(double /*v*/) {
+void WOverview::onRateRatioChange(double /*v*/) {
```
Comment (Holzhaus) [inline src/widget/woverview.cpp]: If you're touching the line anyway, you could also switch to `Q_UNUSED`.

File: src/widget/woverview.cpp [hunk 2]
```diff
@@ -1091,8 +1088,8 @@ void WOverview::paintText(const QString& text, QPainter* pPainter) {
 
 double WOverview::samplePositionToSeconds(double sample) {
     // TODO: replace with rate_ratio in PR #1765
-    double rateRatio = 1.0 + m_pRateDirControl->get() * m_pRateRangeControl->get() * m_pRateSliderControl->get();
-    return sample / m_trackSampleRateControl->get() / mixxx::kEngineChannelCount / rateRatio;
+    return sample / m_trackSampleRateControl->get() / 
+            mixxx::kEngineChannelCount / m_pRateRatioControl->get();
```
Comment (Holzhaus) [inline src/widget/woverview.cpp]: Can't we move this code to a global helper function (maybe `inline`)? I guess there are various places where we need to convert a sample position to seconds.

Also, line 1090  (`// TODO:... `) is obsolete now, so it should be removed. 

Personally, I always find stuff like `a/b/c/d` quite confusing. It's the same as `a/(b*c*d)` right? I prefer that, but that's probably just me.

Comment (daschuer) [reply src/widget/woverview.cpp]: I did not find a similar math in the code. At least I have split the long expression now for a better readability.

**Comment by uklotzde** [comment]
Tests are failing

**Comment by daschuer** [comment]
Now test are working again. The Travis issue is a timeout.

**Comment by uklotzde** [comment]
LGTM

**Pull request merged**